acir/src/native_types/arithmetic.rs

```diff
@@ -149,11 +149,23 @@ impl Expression {
         Ok(expr)
     }
 
-    // TODO: possibly rename, since linear can have one mul_term
```

Comment:
why was this comment removed?

Reply:
because a linear expression does not have a mul_term

Reply:
Agreed, this is my fault for not opening an issue or writing more detail